doc/system_specific_setup.rst

```diff
@@ -1,225 +1,266 @@
 ::
 
    > sudo pip install graphviz
 
 If you just want to use PSyclone then you've installed all you need
-and you're ready to go to the getting-going :ref:`getting-going-run`
-section.
+and you are ready to go to the getting-going :ref:`getting-going-run` section.
+
+.. _dev_setup:
 
 Developer set-up
-++++++++++++++++
+----------------
 
 This section adds software that is used to develop and test
-PSyclone. Note, we assume you have already installed the software
-described in the :ref:`users` section.
+PSyclone. It includes all packages for testing and creation of
+documentation in html and pdf. We assume you have already installed the software
+described in the :ref:`setup_user` section.
 
-pytest
-^^^^^^
+.. _psyclone_from_git:
 
-Install pytest for running python tests
-::
+Installing PSyclone From Git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For development it is recommended to get a copy of PSyclone using git to get 
+access to the latest development version.
 
-   > sudo apt-get install python-pytest
+Installing git for Ubuntu
++++++++++++++++++++++++++
+You need to install the git package::
 
-You can now run the PSyclone python tests:
+    > sudo apg-get install git
 
-.. parsed-literal::
 
-   > cd PSyclone-\ |release|\ /src/psyclone/tests
-   > py.test
+Installing git on OpenSUSE
+++++++++++++++++++++++++++
+You need to install the git package::
+
+    >> sudo zypper --no-recommends install git
 
-Documentation
-^^^^^^^^^^^^^
 
+Cloning PSyclone using git
+++++++++++++++++++++++++++
+Cloning PSyclone from git and setting up your environment is done as follows::
+
+   > cd <PSYCLONEHOME>
+   > git clone https://github.com/stfc/PSyclone.git
+   > cd PSyclone
+   > export PYTHONPATH=`pwd`/src:${PYTHONPATH}
+   > export PATH=`pwd`/bin:${PATH}
+
+You may want to set your python path permanently (e.g. by editing your
+${HOME}/.bashrc file if you run the BASH shell).
+
+Installing documentation tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Install Sphinx for creating PSyclone documentation 
 ::
 
-   > sudo apt-get install python-sphinx
+   > sudo pip install sphinx
 
-You can now build html and latex documentation (but not pdf)
-::
+You can now build html documentation::
 
    > cd doc
    > make html
-   > make latex
 
-Install texlive for the PSyclone pdf documentation.
+The latex package is required to create the pdf documentation
+for PSyclone. Installing the full dependencies can take up several GB,
+the instructions for Ubuntu and OpenSUSE only install a minimal subset.
 
-.. warning:
-    These installs are large. It may be possible to install a subset of texlive-latex-extra but the authors do not know what this subset is.
+Installing LaTeX on Ubuntu
+++++++++++++++++++++++++++
+The following three packages need to be installed to create the pdf documentation.
+It is recommended to install the packages in one ``apt-get`` command, since
+otherwise depending on filesystem unnecessary snapshots might be created that
```

Comment:
s/otherwise depending on filesystem.../otherwise, depending on your filesystem, unnecessary snapshots might be created that take up additional space/